packages/framework/src/date/dateFormatter.ts

```diff
@@ -60,6 +66,8 @@ export const DateFormatter = (): EnsembleFormatter => {
         dayjs(input).diff(currentDate, "month"),
       getYearsDifference: (input: string): number =>
         dayjs(input).diff(currentDate, "year"),
+      humanize: (input: string): string =>
```

Comment:
I think this should be a bit more descriptive - like humanizeDifferenceFromToday? Or, the other option is to make it composable/chainable with the other smaller functions.

Reply:
We'll already be using it composed of `now()` so it's already known that difference is from the current date.
Maybe we should expose same function in `EnsembleFormatter` as well which would take source date as input too. What do you say?

Reply:
I think this is probably fine for now. We can always add an optional second param later. Just to clarify when I say compose - I mean allow the user to compose it with other day js functions, not just internally.